src/Entity/User.php

```diff
@@ -78,6 +80,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
      */
     private Uuid $token;
 
+    /**
+     * @ORM\Column(type="datetime", name="deletedAt")
+     */
+    private ?\DateTime $deletedAt;
```

Comment:
your property hint here is `?\DateTime` but the setter/getter use `?\DateTimeInterface`

Reply:
Right. I'll make correction. Also maker-bundle... definitively I should verify differences...